### DISCUSSION TITLE
Make `WindowNotificationManager` Xaml friendly

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#13612] [Ideas]
## Is your feature request related to a problem? Please describe.
Allow using WindowNotificationManager in `Xaml`

## Describe the solution you'd like

- Mark `WindowNotificationManager` as obsolete
- Create a new `NotificationManager`, it's basically the same except the `Position` and `MaxItems` properties become attached properties. When using `<Usertcontrol NotificationManager.Position="Center"` in Xaml, it is  checked if TopLevel has NotificationManager otherwise it instantiates a new one.

**Comment by timunie** [comment]
In latest master we relaxed the WNM already to make usage in xaml easier. I don't know if we want to invest more into it as I think we should instead invest our time into: 
- native notifications API for each platform 
- FluentAvalonia like ContentControl / DialogHost solution 
- with 2), the Notification Manager can be extended to make use of the DialogHost probably.

**Reply by maxkatz6** [reply]
https://github.com/AvaloniaUI/Avalonia/blob/0567a6dbe7dc122ba90e019320708e0cbc51b635/samples/ControlCatalog/Pages/NotificationsPage.xaml#L25

**Reply by workgroupengineering** [reply]
I miss it. Tnx
